stable/pipelines/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: pipelines
 home: https://jfrog.com/pipelines/
-version: 1.1.5
-appVersion: 1.4.7
+version: 1.1.6
```

Comment:
I think you should bump the minor version in the chart version

Reply:
Done